asllib/TopoSort.ml

```diff
@@ -192,6 +192,7 @@ module ASTFold = struct
     | D_Func { name; _ } | D_GlobalStorage { name; _ } | D_TypeDecl (name, _, _)
       ->
         name
+    | D_Pragma _ -> "__pragmas"
```

Comment:
Since you're separating global pragmas from the rest of the global declarations, they shouldn't appear here, so perhaps `assert false`?